Libraries/LibLine/Editor.cpp

```diff
@@ -33,8 +33,9 @@
 
 namespace Line {
 
-Editor::Editor()
+Editor::Editor(bool lazy)
 {
+    m_is_lazy = lazy;
```

Comment:
What does lazy mean in this context?

Reply:
Whether it will draw lazily (only whatever is needed)
I guess I'll rename it to always_refresh